### PR TITLE
web/satellite: internal server error template added, errors separated

### PR DESCRIPTION
**Pull request by Qweder93**
What: web/satellite: internal server error template added, 500 error separated from 404

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/console/consoleweb/server.go [hunk 1]
```diff
@@ -420,14 +421,19 @@ func (server *Server) cancelPasswordRecoveryHandler(w http.ResponseWriter, r *ht
 }
 
 func (server *Server) serveError(w http.ResponseWriter, r *http.Request, status int) {
```
Comment (rikysya) [inline satellite/console/consoleweb/server.go]: request not needed

File: web/satellite/static/errors/500.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!--Copyright (C) 2019 Storj Labs, Inc.-->
+<!--See LICENSE for copying information.-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Tardigrade Satellite - Mars</title>
+
```
Comment (rikysya) [inline web/satellite/static/errors/500.html]: remove empty line

File: satellite/console/consoleweb/server.go [hunk 3]
```diff
@@ -420,14 +421,19 @@ func (server *Server) cancelPasswordRecoveryHandler(w http.ResponseWriter, r *ht
...
-	// TODO: show different error pages depend on status
-	// F.e. switch(status)
-	//      case http.StatusNotFound: server.executeTemplate(w, r, notFound, nil)
-	//      case http.StatusInternalServerError: server.executeTemplate(w, r, internalError, nil)
 	w.WriteHeader(status)
 
-	if err := server.templates.pageNotFound.Execute(w, nil); err != nil {
-		server.log.Error("error occurred in console/server", zap.Error(err))
+	switch status {
+	case http.StatusNotFound:
```
Comment (crawter) [inline satellite/console/consoleweb/server.go]: let's make this case default one

File: web/satellite/static/errors/500.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
+/* Copyright (C) 2019 Storj Labs, Inc. */
+/* See LICENSE for copying information. */
+
+.error-container {
```
Comment (crawter) [inline web/satellite/static/errors/500.css]: you should double check if this css file corresponds our new styles style guide
unfortunately, this file won't be checked by our linter

File: satellite/console/consoleweb/server.go [hunk 5]
```diff
@@ -85,12 +85,13 @@ type Server struct {
...
 	schema    graphql.Schema
 	templates struct {
-		index         *template.Template
-		pageNotFound  *template.Template
-		usageReport   *template.Template
-		resetPassword *template.Template
-		success       *template.Template
-		activated     *template.Template
+		index               *template.Template
+		pageNotFound        *template.Template
```
Comment (crawter) [inline satellite/console/consoleweb/server.go]: notFound will be better

File: satellite/console/consoleweb/server.go [hunk 6]
```diff
@@ -348,13 +349,13 @@ func (server *Server) accountActivationHandler(w http.ResponseWriter, r *http.Re
...
 		// TODO: when new error pages will be created - change http.StatusNotFound on appropriate one
-		server.serveError(w, r, http.StatusNotFound)
+		server.serveError(w, http.StatusNotFound)
 		return
 	}
 
 	if err = server.templates.activated.Execute(w, nil); err != nil {
 		server.log.Error("satellite/console/server: account activated template could not be executed", zap.Error(err))
-		server.serveError(w, r, http.StatusNotFound)
+		server.serveError(w, http.StatusNotFound)
```
Comment (crawter) [inline satellite/console/consoleweb/server.go]: server.log.Error("satellite/console/server: account activated template could not be executed", zap.Error(Error.Wrap(err)))

**Pull request merged**